requirements.txt

```diff
@@ -9,3 +9,4 @@ mongomock==3.19.0
 django==3.0.7
 elasticsearch-dsl==7.1.0
 Jinja2==2.11.1
+typing-extensions==3.7.4.1
```

Comment:
Can we keep the notion of "only for python<3.8" or is that no longer possible?

Reply:
Again, just for the CI, so in the actual install, it will use what is in setup.py